webpack/assets/javascripts/foreman_tools.js

```diff
@@ -80,3 +81,22 @@ export function initTypeAheadSelect(input) {
     width: '400px'
   });
 }
+
+// send get forms through turbolinks
+export function tableUpdated(element) {
+  let uri = new URI(window.location.href);
+  let search;
+
+  if (element !== undefined) {
```

Comment:
maybe you can use a ternary operator here:
```
search = (element !== undefined) ? $(element).find('.autocomplete-input') : $('#search')
```
Nevertheless, I am not sure how readable will it be.